lib/fortnox/api/models/document_row.rb

```diff
@@ -0,0 +1,79 @@
+require "fortnox/api/models/base"
+
+module Fortnox
+  module API
+    module Model
+      module DocumentRow
+        DISCOUNT_TYPES = %w(AMOUNT PERCENT)
+        HOUSE_WORK_TYPES = %w(CONSTRUCTION ELECTRICITY GLASSMETALWORK GROUNDDRAINAGEWORK MASONRY PAINTINGWALLPAPERING HVAC CLEANING TEXTILECLOTHING COOKING SNOWPLOWING GARDENING BABYSITTING OTHERCARE TUTORING OTHERCOSTS)
+
+        def discount_type=( raw_discount_type )
+          discount_type = raw_discount_type.upcase
+          super discount_type if DISCOUNT_TYPES.include? discount_type
+        end
+
+        def house_work_type=( raw_house_work_type )
```

Comment:
Look at the attributes for these. If you need to do data mangling it's probably worth making it into an attribute. But skip it for now, the attributes implementation will be a bit different using dry-types...